js/script.js

```diff
@@ -0,0 +1,185 @@
+let allTasks = [];
+let valueInput = '';
+let input = null;
+let content;
+let activeEditTask = null;
+
+const buttonTasksAdd = async () => {
+  if (!valueInput.trim()) { return; }
+
+  allTasks.push({
+    text: valueInput.trim(),
+    isCheck: false,
+  });
+
+  const resp = await fetch('http://localhost:8000/createTask', {

```

Comment:
http://localhost:8000 is repeated a lot, make it a global variable
for example "url"